### DISCUSSION TITLE
Proposal: new defined format: "semver"

### DISCUSSION THREAD

**Discussion by GlennButera** [json-schema-org/community#165] [Specification]
Long time listener, first time caller, and an all-round big fan of JSON Schema.

I'm wondering if the community would be open to discussing the addition of another "Defined Format" to the list of formats in the JSON Schema Validation Specification.  Semantic Versioning is a commonly used version string.  It would be very helpful to be able to denote a JSON Schema of type string to have the format of "semver".  Although "semver" does not have an official RFC, they do have a spec for valid Semantic Version strings on [their website](https://semver.org/).  I understand that we can continue to utilize the ability to use custom format attributes, so this isn't necessarily a high priority.

**Comment by gregsdennis** [comment]
This had been proposed and declined already.  Please see https://github.com/json-schema-org/json-schema-spec/issues/1108 for discussion and reasoning.

Thanks for the proposal though.  We appreciate your involvement and input.
